bigchaindb/upsert_validator/validator_election.py

```diff
@@ -145,3 +150,79 @@ def create(cls, tx_signers, recipients, metadata=None, asset=None):
     @classmethod
     def transfer(cls, tx_signers, recipients, metadata=None, asset=None):
         raise NotImplementedError
+
+    @classmethod
+    def to_public_key(cls, election_id):
+        return base58.b58encode(bytes.fromhex(election_id))
+
+    @classmethod
+    def count_votes(cls, election_pk, txns, getter=getattr):
```

Comment:
When do we need to change `getter`?

Reply:
The function can be called with either `dict` or `ValidatorElectionVote` class objects in `txns` list, so different `getattr`s are needed

Reply:
I saw @ldmberman's comment after writing mine.

I don't like this solution much because we are delegating polymorphism to `getter`. I agree with @ldmberman that ideally we should deal only with dictionaries, and avoid custom objects.

Another idea is to have the class `Transaction` implement `__getitem__`.

Maybe we can keep it like this for now.

I'd still change `txns` to `transactions`.